src/core/symbol/SymbolKeyPair.ts

```diff
@@ -70,6 +71,8 @@ export class SymbolKeyPair extends KeyPair {
      * @returns true if the signature is verifiable, false otherwise.
      */
     public verify(data: Uint8Array, signature: Uint8Array): boolean {
+        if (!this.IsCanonicalS(signature.slice(signature.length / 2))) return false;
```

Comment:
(minor) not sure about what formatting you have set for the project, but it's generally easier to read if statement body is on a separate line